src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(str -> Arrays.stream(str.split(",")))
```

Comment:
do not use such naming(str, num), how to guess what it stores?
```suggestion
                .flatMap(str -> Arrays.stream(str.split(",")))
```
```suggestion
                .flatMap(lineNumbers -> Arrays.stream(lineNumbers.split(DELIMITER)))
```